apps/watch/src/components/NewContentPage/Quiz/Slides/Q1/Q1.tsx

```
@@ -0,0 +1,191 @@
+              label={option.label}
+              imageUrl={option.imageUrl}
+              isSelected={selectedIdx === idx}
+              onClick={handleOptionClick}
+              onHover={handleHover}
+              t={t}
+            />
+          ))}
+        </div>
+      </div>
+    </div>
+  )
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)